src/applications/vaos/components/ConfirmedAppointmentListItem.jsx

```diff
@@ -69,9 +73,9 @@ export default function ConfirmedAppointmentListItem({
       </div>
       <h3 className="vaos-appts__date-time vads-u-font-size--h3 vads-u-margin-x--0">
         <AppointmentDateTime
-          appointmentDate={appointment.appointmentDate}
-          timezone={appointment.timeZone}
-          facilityId={appointment.facilityId}
+          appointmentDate={moment.parseZone(appointment.start)}
```

Comment:
Fixed timezones for VA appointments.  They weren't getting passed the correct facilityId to grab the timezone.

In regards to the green bar, looks like you tested it right very close to the appointment time.  This appointment is within the "future" threshold of 60 minutes ago so it still appeared on the future list, but `appt.vaos.isPastAppointment` was set to true because it wasn't taking this threshold into account.  I updated isPastAppointment() to take this threshold into account and added unit tests.